controller/model/enrollment_mod_ca.go

```diff
@@ -189,14 +189,14 @@ func (module *EnrollModuleCa) completeCertAuthenticatorEnrollment(log *logrus.En
 		},
 	}
 
-	_, authenticatorId, err := module.env.GetManagers().Identity.CreateWithAuthenticator(identity, newAuthenticator, context.GetChangeContext())
+	_, authenticatorIds, err := module.env.GetManagers().Identity.CreateWithAuthenticators(identity, []*Authenticator{newAuthenticator}, context.GetChangeContext())
 
 	if err != nil {
 		log.WithError(err).Error("failed to create identity with authenticator")
 		return nil, err
 	}
 
```

Comment:
Any chance of a panic here (can the authenticator list ever be empty)?

Reply:
Technically, that is fine. It is just a create at that point.